lib/charms/postgresql_k8s/v1/postgresql.py

```diff
@@ -1058,6 +1060,11 @@ def set_up_database(self, temp_location: Optional[str] = None) -> None:
             cursor = connection.cursor()
 
             if temp_location is not None:
+                # Fix permissions on the temporary tablespace location when a reboot happens and tmpfs is being used.
+                user = pwd.getpwnam("_daemon_")
+                os.chown(temp_location, uid=user.pw_uid, gid=user.pw_gid)
+                os.chmod(temp_location, 0o700)
```

Comment:
Also, it is 750 [there](https://github.com/canonical/postgresql-operator/blob/cc1a6239cd77e84951438db0e51cbf68bc45f953/src/relations/async_replication.py#L663)

Are we consistent?